src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2FilteringEngine.java

```diff
@@ -202,103 +193,14 @@ private void applyDuplicatedAltReadFilter(final M2FiltersArgumentCollection MTFA
         }
     }
 
-
```

Comment:
this code belongs in my other branch but somehow made it in here so I simply removed it